src/main/java/com/gulteking/mqttbackendserver/serviceimpl/MqttSubscriberImpl.java

```diff
@@ -121,6 +113,7 @@ public void subscribeMessage(String topic) {
         try {
             this.mqttClient.subscribe(topic, this.qos);
         } catch (MqttException me) {
+            logger.error("ERROR", me);
             System.out.println("Not able to Read Topic  " + topic);
```

Comment:
if using Logger then can remove all SOP statements and move relevant messages to Logs only.